_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -4,14 +4,27 @@ title: Server-Side Rendering
 
 In Spartacus, server-side rendering allows you to render static versions of pages on the server side. This speeds up response times, assists with SEO, and allows the application to render more quickly. After Angular has bootstrapped, users of your site will have the full experience.
 
-***
+---
 
 **Table of Contents**
 
 - This will become a table of contents (this text will be scrapped).
-{:toc}
+  {:toc}
+
+---
+
+## Which pages to server-side render?
+
+It is generally recommended to server-side render pages which don't change very frequently and don't contain a notion of personalization. In the e-commerce world, this usually means that you shouldn't SSR the following pages:
+
+- homepage - often contains personalized content (e.g. products, carousels, etc.)
+- user profile / account management pages / cart / checkout - very user-oriented pages
+
+These are the pages that you can consider to server-side render, depending on yor needs:
 
-***
+- faq, about us - these pages rarely change
+- product details pages - it usually doesn't change often. But, in some cases some data might be changing frequently (price, stock, etc.). Therefore, if caching the SSR responses, please make sure to have proper cache invalidation strategy in place.
```

Comment:
Another recommended approach is to allow the pages to be SSred and cached, but have a dynamic service to update stock and pricing after page load, depending on how often they change based on your particular business rules

Reply:
Correct, and this is something we'll have to support in the future, but we're not there yet.